packages/private-build-infra/addon/index.ts

```diff
@@ -0,0 +1,22 @@
+/**
+  @module @ember-data/private-build-infra
+*/
+
+import { has } from 'require';
+import POSSIBLE_PACKAGES from './available-packages';
+
+type PackageSettings = Record<keyof typeof POSSIBLE_PACKAGES, boolean>;
+
+const PACKAGES = Object.keys(POSSIBLE_PACKAGES).reduce((obj, name) => {
```

Comment:
`PACKAGES` isn't exported here, I'd prefer to make a simpler method and invoke it just below

Maybe something like this:

https://github.com/emberjs/ember.js/blob/892ca20980c67aaef247bdd5c69263f81522d7e1/packages/%40ember/canary-features/index.ts#L63-L77

Reply:
cleaned up